NEWS.md

```diff
@@ -3,6 +3,8 @@
 
 * `vars_select()` ignores vectors with only zeros (#82). 
 
+* `matches()` has new argument `perl` to allow for Perl-like regular expressions (requested by @GegznaV, #71)
```

Comment:
Can you please follow the style in https://style.tidyverse.org/news.html#bullets? (i.e. it should mention you as the PR writer, not the original issue creator)

Reply:
thanks, done!